core/src/main/java/org/apache/calcite/rel/rules/SubQueryRemoveRule.java

```diff
@@ -405,8 +405,9 @@ private static RexNode rewriteSome(RexSubQuery e, Set<CorrelationId> variablesSe
         //   from emp group by name) as q on e.name = q.name
         builder.push(e.rel)
             .aggregate(builder.groupKey(),
-                builder.count(true, "c"),
-                builder.count(true, "d", builder.field(0)),
+                builder.count(false, "c"),
```

Comment:
is the comment above describing the rewrite accurate?

Reply:
oops, forgot to update it. nice catch

Reply:
is this enough? where is dd used?

Reply:
@NobiGo did you happen to author this code?
Then perhaps you are the most suitable person to approve.

Reply:
I will check this.